pep-0572.rst

```diff
@@ -255,7 +262,8 @@ Relative precedence of ``:=``
 
 The ``:=`` operator groups more tightly than a comma in all syntactic
 positions where it is legal, but less tightly than all operators,
```

Comment:
s/all operators/all other operators/ ?  That change would seem to be in the spirit of the OP's complaints.

Reply:
According to the table at https://docs.python.org/3/reference/expressions.html#operator-precedence the comma is not an operator in Python. It's always a syntactic construct. I suppose you could argue that it's an operator taking two or more operands, and returns a tuple containing them, but that's pushing the definition a bit.

Reply:
Right, comma isn't "really" an operator, but pretending otherwise is common in explanatory text.  The idea is that the RHS of `:=` goes on until you hit a comma or unmatched right paren/bracket/brace.  The fiction is useful because specifying the actual grammar is full of details that - while necessary for the implementation - aren't really illuminating.

Reply:
Okay. Then let's go with "other"; it'll be technically correct in that "all other operators" means "everything except `:=`", and colloquially correct in that it means "everything except the comma".

Reply:
The "all other operators" change is already part of https://github.com/python/peps/pull/718.